### DISCUSSION TITLE
Please make an arm64 release.

### DISCUSSION THREAD

**Discussion by dtasada** [mat/besticon#86] [General]
I've attempted to compile it for arm, but i can't quite get it to work. Though, I know nothing about golang, so it's probably my fault. I was wondering if you could take the time to compile it for arm if possible. I tried the docker images, but those also don't seem to work.

**Comment by mat** [comment]
I added support for building arm64 binaries, please have a look: https://github.com/mat/besticon/releases/tag/v3.16.0

**Reply by dtasada** [reply]
Thank you, it works now!
